src/device/dap-wrapper.ts

```diff
@@ -112,7 +112,10 @@ export class DAPWrapper {
   }
 
   async startSerial(listener: (data: string) => void): Promise<void> {
-    await this.daplink.setSerialBaudrate(115200);
+    const currentBaud = await this.daplink.getSerialBaudrate();
+    if (currentBaud != 115200) {
```

Comment:
Please use JavaScript's strict equality operator.
```suggestion
    if (currentBaud !== 115200) {
```